www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -283,8 +283,30 @@
         )
     );
 
+    $durationScale = array(
+        'd' => 86400,
+        'h' => 3600,
+        'm' => 60,
+        's' => 1,
+    );
     $defaultDuration = 7200;
+    $defaultScale = 's';
+    if (isset($centreon->optGen['monitoring_dwt_duration']) &&
+        $centreon->optGen['monitoring_dwt_duration']
+    ) {
+        $defaultDuration = $centreon->optGen['monitoring_dwt_duration'];
+        if (isset($centreon->optGen['monitoring_dwt_duration_scale']) &&
+            $centreon->optGen['monitoring_dwt_duration_scale']
+        ) {
+            $defaultScale = $centreon->optGen['monitoring_dwt_duration_scale'];
+        }
+    }
     $form->setDefaults(array('duration' => $defaultDuration));
+    $form->setDefaults(
+        array(
+            "alternativeDateEnd" => $centreonGMT->getDate("Y-m-d", time() + $defaultDuration * $durationScale[$defaultScale], $gmt)
```

Comment:
The variable shouldn't hardcode the localization format.
We add moment.js to convert BE timestamp to a localized FE one

Reply:
This end date is `GMT`, to feed the FE datepicker, which then automatically localizes it.
As the start date.

Reply:
Done, removed.